### DISCUSSION TITLE
Issue SPI API ORDER You exceeded your quota for the requested resource

### DISCUSSION THREAD

**Discussion by jaymingehlot** [amzn/selling-partner-api-models#3729] [General]
Hello guys my self Jaymin,

I want to ask something for WebHooks for sp-orders to sort out the most of the Amazon SP Api issue.

My issue is I don't have any idea to which time i want to hit SP Api for new created orders in this case if amazon provide us for webhooks for new created for or status change on particular order in that can want need to hit api on every 5 minutes or 30 minutes.

If you any best solution please give  me I'm facing too much issue related SP orders api.

Everytime I saw below error: 

  ```
[429] "errors": [
    {
      "code": "QuotaExceeded",
      "message": "You exceeded your quota for the requested resource.",
      "details": ""
    }
  ]
}
```

**Comment by MeenaAmz** [comment]
We recommend subscribing to [order_change notification ](https://developer-docs.amazon.com/sp-api/docs/notifications-api-v1-use-case-guide#order_change) which will provide information about whenever there is an important change in the order like order status changes and buyer requested cancelations. Then based on the notifications, you can call orders API only when needed.
